### PR TITLE
DOCS-4962 API Tests, Browser Tests, and Advanced Options for Browser Tests Doc Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Replaces individual sections for variables in API Tests, Browser Tests, and Advanced Options for Browser Tests doc pages with reusable short code. Updates some screenshots and videos.

### Motivation

DOCS-4962; discussion in a Google Doc

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by rtrieu — APPROVED** [review]
Looks great!

File: layouts/shortcodes/synthetics-variables.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+&#x7b;&#x7b; alphanumeric(n) &#x7d;&#x7d;
+: Generates an alphanumeric string with `n` characters.
+
+&#x7b;&#x7b; uuid &#x7d;&#x7d;
+: Generates a version 4 universally unique identifier (UUID).
+
+&#x7b;&#x7b; date(n unit, format) &#x7d;&#x7d;
+: Generates a date in one of Datadog's accepted formats with a value corresponding to the UTC date the test is initiated at + or - `n` units.
+
+&#x7b;&#x7b;timestamp(n, unit)&#x7d;&#x7d;
```
Comment (HugoPct) [inline layouts/shortcodes/synthetics-variables.md:22]:
```suggestion
&#x7b;&#x7b; timestamp(n, unit) &#x7d;&#x7d;
```

File: content/en/synthetics/settings/_index.md [hunk 2]
```diff
@@ -118,7 +118,18 @@ For more information about TOTP-based MFA in a browser test, see [TOTPs For Mult
 
 Once created, global variables can be used in all Synthetic tests. To import your global variables into your test, click **+ Variables**, type `{{` in a field you want to add the variable, and select your global variable. 
 
-For more information, see the [HTTP test][8], [Multistep API test][9], [Browser test configuration][10], and [Steps documentation][16].
+To update the value of an existing global variable using [Terraform][17], create a global variable with the same name, description, and tags, but different value.
```
Comment (HugoPct) [inline content/en/synthetics/settings/_index.md:121]: I do not think we should talk about terraform here. It adds complexity and it is the only place in the all doc we have it. I would recommend we keep TF documentation on the TF repository

**Review by HugoPct — COMMENTED** [review]
left 2 comments otherwise good to go!
great sections on subtests!!

File: content/en/synthetics/settings/_index.md [hunk 3]
```diff
@@ -118,7 +118,18 @@ For more information about TOTP-based MFA in a browser test, see [TOTPs For Mult
...
+To update the value of an existing global variable using [Terraform][17], create a global variable with the same name, description, and tags, but different value. 
+
+```json
+resource "datadog_synthetics_global_variable" "test_variable" {
+  name        = "EXAMPLE_VARIABLE"
+  description = "Description of the variable"
+  tags        = ["foo:bar", "env:test"]
+  value       = "NEW-VALUE"
+}
+```
```
Comment (alai97) [inline content/en/synthetics/settings/_index.md:130]:
```suggestion
```

File: content/en/synthetics/settings/_index.md [hunk 4]
```diff
@@ -210,3 +221,4 @@ If you are using the [custom role feature][12], add your user to any custom role
 [14]: /synthetics/guide/explore-rum-through-synthetics/
 [15]: /synthetics/apm/#prerequisites
 [16]: /synthetics/browser_tests/actions/#use-variables
+[17]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/synthetics_global_variable
```
Comment (alai97) [inline content/en/synthetics/settings/_index.md:224]:
```suggestion
```

**Pull request merged**